README.md

```diff
@@ -212,26 +220,27 @@ http://kovan.dex.top/v1/market
 
 ## GetPairsByCash
 
-This API for getting realtime  trade infomation of  all pair of specific cashtoken such as ETH. This API data can get all tradepair realtime trade infomation.
+Get real-time trade information of all pair of specific cash token such as ETH. 
```

Comment:
Get the real-time trading information of all available trading pairs of the specified cash token (e.g. "ETH").
Remove "This API data can get all trade pair real-time trade information."